src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="navContainer">
```

Comment:
The name of the class does not correspond to the BEM methodology, please check all the names of the classes.